storage/internal/benchmarks/README.md

```diff
@@ -31,6 +31,7 @@ This runs 1000 iterations on 512kib to 2Gib files in the background, sending out
 | -connection_pool_size | GRPC connection pool size | any positive integer | 4 |
 | -force_garbage_collection | whether to force garbage collection <br> before every write or read benchmark |  `true` or `false` (present/not present) | `false` |
 | -timeout | timeout (maximum time running benchmarks) <br> the program may run for longer while it finishes running processes | any [time.Duration](https://pkg.go.dev/time#Duration) | `1h` |
-| -directory | upload and download an entire directory | `true`/`false` | `false` |
+| -workload | `1` will run a w1r3 (write 1 read 3) benchmark <br> `6` will run a benchmark uploading and downloading a directory (once each) | `1` or `6` | `1` |
+| -objects_per_directory | total number of objects in a directory; <br> only applies to workload 6 | any positive integer | `1000` |
```

Comment:
Maybe just `-num_objects`? It should be clear that there is one directory and this controls the total number of objects uploaded/downloads.

Reply:
I wanted to make it clear that this parameter only applies to directories. `-num_objects` might be confused with `-samples`. If you think that isn't an issue, I'm fine with changing it to `num_objects`